ConstructBinaryTreefromInorderandPostorderTraversal.java

```diff
@@ -0,0 +1,55 @@
+package s30Coding;
+import java.util.*;
+
+//Time Complexity :- O(n)
+
+//Space Complexity :- O(n)
```

Comment:
We store the tree, but it is the result. So, tighter upper bound for space complexity here would be O(H), which is recursive stack space